eng/create-baridtag.ps1

```diff
@@ -0,0 +1,24 @@
+Param(
+  [Parameter(Mandatory=$true)][string] $azdoToken,
+  [Parameter(Mandatory=$true)][string] $barToken,
+  [Parameter(Mandatory=$true)][string] $githubPAT
+)
+
+set-strictmode -version 2.0
+$ErrorActionPreference = 'Stop'
+
+. $PSScriptRoot\common\tools.ps1
+$darc = & "$PSScriptRoot\validation\get-darc.ps1"
+
+$global:azdoToken = $azdoToken
+$global:githubPAT = $githubPAT
+$global:barToken = $barToken
+
+$global:arcadeSdkPackageName = 'Microsoft.DotNet.Arcade.Sdk'
+$global:arcadeSdkVersion = $GlobalJson.'msbuild-sdks'.$global:arcadeSdkPackageName
+$jsonAsset = & $darc get-asset --name $global:arcadeSdkPackageName --version $global:arcadeSdkVersion --github-pat $global:githubPAT --azdev-pat $global:azdoToken --password $global:bartoken --output-format json | convertFrom-Json
```

Comment:
[nit] This isn't really json right? you're already deserializing it to an object.